modules/rubiconBidAdapter.js

```diff
@@ -14,6 +14,19 @@ export const FASTLANE_ENDPOINT = '//fastlane.rubiconproject.com/a/api/fastlane.j
 export const VIDEO_ENDPOINT = '//prebid-server.rubiconproject.com/openrtb2/auction';
 export const SYNC_ENDPOINT = 'https://eus.rubiconproject.com/usync.html';
 
+const DIGITRUST_PROP_NAMES = {
+  FASTLANE: {
+    id: 'dt.id',
+    keyv: 'dt.keyv',
+    pref: 'dt.pref'
+  },
+  PREBID_SERVER: {
+    id: 'id',
+    keyv: 'keyv',
+    pref: 'pref'
```

Comment:
please drop 'pref' from both of these - if privacy.optout=true (pref=0), no DT value is sent.

Reply:
Does this apply to both OpenRTB and Rubicon AE? `pref` property shouldn't be sent to either one?

Reply:
It's possible that AE needs these things. :-(

So leave the pref=0 going to AE. Sorry for the back and forth.

Reply:
Removed sending `pref` to OpenRTB in latest commit